.github/workflows/ci.yml

```diff
@@ -37,6 +37,14 @@ jobs:
         uses: actions/checkout@v2
       - name: "Run TypoScript lint"
         uses: TYPO3-Continuous-Integration/TYPO3-CI-Typoscript-Lint@v1
+  codestyle-check:
+    name: "Codestyle check"
```

Comment:
Please let's rename this job (and the new action and the Composer action) to "PHP CS Fixer" (`ci:php:cs-fixer`) to make it more clear what tool this is about.

Reply:
I've done one part in #227.